shogun_web.py

```diff
@@ -105,9 +105,9 @@ def sitemap():
 
 
 @app.route('/docs/<path:filename>')
-def docs_static(filename):
-    print app.root_path
-    return send_from_directory(DOCS_SUBMODULE_DIR, filename)
+def docs_static(filename, template='doc.html'):
+    doc_content = get_markuped(filename.lower(), "docs/%s" % filename)
```

Comment:
prefer the newer `.format`

Reply:
Do you mean something like this  `"docs/{}".format(filename)`?

Reply:
yes

Reply:
fixed at https://github.com/shogun-toolbox/shogun-web2/pull/43/commits/5d424a5d7589b768895d6c40395a85d2caad59e5